src/ripple/app/main/Application.cpp

```diff
@@ -1027,12 +1034,15 @@ void ApplicationImp::setup()
                 startUp == Config::LOAD_FILE ||
                 startUp == Config::REPLAY)
     {
-        JLOG(m_journal.info()) << "Loading specified Ledger";
+        JLOG(m_journal.info()) <<
+            "Loading specified Ledger";
 
         if (!loadOldLedger (config_->START_LEDGER,
                             startUp == Config::REPLAY,
                             startUp == Config::LOAD_FILE))
         {
+            JLOG(m_journal.error()) <<
+                "The specified ledger could not be loaded.";
```

Comment:
What do you think of using `"Ledger " << config_->START_LEDGER << " could not be loaded."`?